### ISSUE TITLE
Add a dismissable usability warning to dashboard

### ISSUE THREAD

**Issue by malini** [thunderbird/appointment#684]
**Is your feature request related to a problem? Please describe.**
During the public beta, we are using a legacy Zoom app integration since we're in the middle of getting the requirements in place for upgrading to the latest version. In the meantime, users may run into an issue where their meetings won't have the zoom meeting link due to session token issues.

**Describe the solution you'd like**
We should add a dismissable usability warning here:
![Screenshot 2024-09-24 at 12 10 29 PM](https://github.com/user-attachments/assets/2646a566-6283-49fc-a1ae-2620a7a024a3)

with details of issues users may have during the beta:
* emails may go to spam, so check spam folder
* If Zoom meetings don't appear in the invite, disconnect and reconnect your zoom account in the [Connected Accounts](https://appointment.day/settings/connectedAccounts) section of the settings.
* If you experience other issues, please use the [contact us](https://appointment.day/contact) form, or join us on our [Matrix Channel](https://matrix.to/#/#tb-services:mozilla.org)

**Comment by MelissaAutumn** [comment]
We have an email that goes out to the schedule owner in case a zoom link fails:

<img width="639" alt="image" src="https://github.com/user-attachments/assets/d4951c21-8a00-4958-8dd5-d0874ee977fc">

So we notify them, but a warning helps. (And maybe adding it to the email as well.)

And yep that all sounds good otherwise!

**Comment by inattee** [comment]
The last two bullet points sound great.  My suggestion would be to only slightly tweak the first two lines:

As we are in beta, a few known issues are currently being addressed:

- Please check your spam folder as some emails might end up there

**Comment by malini** [comment]
great, thanks @inattee!

**Comment by MelissaAutumn** [comment]
How's this look?
<img width="1310" alt="image" src="https://github.com/user-attachments/assets/d9c541f0-4470-4c28-8371-9092024b0582">

cc @laurelterlesky @malini

**Comment by laurelterlesky** [comment]
Yes that works for me. My only nit picks are aligning the information icon to the top left and the dismiss btn to the top right (you can also just use the close X in a circle too)

**Comment by MelissaAutumn** [comment]
I'll have to find the icon figma file again, but with edits:
<img width="1307" alt="image" src="https://github.com/user-attachments/assets/892e87eb-dbf9-488a-bc94-1bdcb036bdca">

**Comment by laurelterlesky** [comment]
Beauty. Ship it :)
